hw/ip/kmac/dv/kmac_base_sim_cfg.hjson

```diff
@@ -140,6 +139,15 @@
       name: "{variant}_app"
       uvm_test_seq: kmac_app_vseq
     }
+    {
+      name: kmac_stress_all
+      run_opts: ["+test_timeout_ns=10000000000",
```

Comment:
maybe add some "_" in between 0s to it is easier to read?

Reply:
good suggestion, thanks! let me add this.